### PR TITLE
[stable/kong] bump up dependencies

### PR DESCRIPTION
**Pull request by hbagdi**
Signed-off-by: Harry Bagdi <harrybagdi@gmail.com>

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] title of the PR contains starts with chart name e.g. `[stable/chart]`

### CODE REVIEW

File: stable/kong/requirements.lock [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 - name: postgresql
   repository: https://kubernetes-charts.storage.googleapis.com/
-  version: 3.9.1
+  version: 3.18.3
 - name: cassandra
   repository: https://kubernetes-charts-incubator.storage.googleapis.com/
-  version: 0.10.2
-digest: sha256:ed6abe23258b9eb42a55b21ff68d5df7367e04d0c1de588442f6a91a067616a9
-generated: 2019-01-09T12:04:14.047647677-08:00
+  version: 0.10.5
```
Comment (shashiranjan84) [inline stable/kong/requirements.lock:7]: why https://github.com/helm/charts/pull/13469/files#diff-509c05d7eb2ab1aaa0709017c4b59a61R7 pointing to `0.10.2` ?

Comment (hbagdi) [reply stable/kong/requirements.lock:7]: fixed

**Review by shashiranjan84 — APPROVED** [review]
/lgtm

**Pull request merged**